modelconverter/utils/onnx_tools.py

```diff
@@ -176,3 +181,975 @@ def onnx_attach_normalization_to_inputs(
 
     onnx.save(model, str(save_path))
     return save_path
+
+
+@dataclass
+class ONNXModifier:
+    """ONNX model modifier class to optimize and modify the ONNX model.
+
+    Attributes:
+    ----------
+    model_path : Path
+        Path to the base ONNX model
+    output_path : Path
+        Path to save the modified ONNX model
+    """
+
+    model_path: Path
+    output_path: Path
+
+    def __post_init__(self):
+        self.load_onnx()
+
+    def load_onnx(self) -> None:
+        """Load the ONNX model and store it as onnx.ModelProto and
+        onnx_graphsurgeon.GraphSurgeon graph."""
+
+        logger.info(f"Loading model: {self.model_path.stem}")
+
+        # Load the ONNX model
```

Comment:
Most/all of the inline comments can be deleted. It's pretty clear what's happening from the code.